components/aws/sagemaker/hyperparameter_tuning/component.yaml

```diff
@@ -189,11 +189,10 @@ implementation:
       --checkpoint_config, {inputValue: checkpoint_config},
       --warm_start_type, {inputValue: warm_start_type},
       --parent_hpo_jobs, {inputValue: parent_hpo_jobs},
-      --tags, {inputValue: tags}
-    ]
-    fileOutputs:
-      hpo_job_name: /tmp/hpo_job_name.txt
-      model_artifact_url: /tmp/model_artifact_url.txt
-      best_job_name: /tmp/best_job_name.txt
-      best_hyperparameters: /tmp/best_hyperparameters.txt
-      training_image: /tmp/training_image.txt
+      --tags, {inputValue: tags},
+      --hpo_job_name_output_path, {outputPath: hpo_job_name},
+      --model_artifact_url_output_path, {outputPath: model_artifact_url},
+      --best_job_name_output_path, {outputPath: best_job_name},
+      --best_hyperparameters_output_path, {outputPath: best_hyperparameters},
+      --training_image_output_path, {outputPath: training_image}
```

Comment:
Thought: There should be a way to specify a common custom folder for all outputs instead of having to change it for each one.

Reply:
Users aren't able to override these values and KFP always injects values into it anyway, so the default values are practically useless. I was just following the convention of the example Alexey posted.

Reply:
Yea I know

Reply:
Each output has separate path so that it's possible to separately pass outputs to downstream components. Imagine if the data passing was implemented as volume mounting. If all output values were in the same directory, then the downstream components will be receiving all output files even when asking for a specific one.

Reply:
That makes sense, thank you!